3rdparty/python/requirements.txt

```diff
@@ -20,6 +20,7 @@ pathspec==0.8.0
 pex==2.1.16
 psutil==5.7.0
 pystache==0.5.4
+pytest>=6.0.1,<6.1
```

Comment:
A comment here referencing that this needs to be kept in sync with the option value would be good. Ideally the ability to override the pytest version would come from an `target_option` referring to this type to remove the redundancy.